suite-native/module-authenticity-checks/src/components/FirmwareAuthenticityCheckFailModalContent.tsx

```diff
@@ -1,56 +1,34 @@
 import { useDispatch, useSelector } from 'react-redux';
 
-import { useNavigation } from '@react-navigation/native';
-
 import { deviceActions, selectSelectedDevice } from '@suite-common/wallet-core';
 import { Button } from '@suite-native/atoms';
 import { Translation } from '@suite-native/intl';
-import {
-    AppTabsRoutes,
-    HomeStackRoutes,
-    RootStackParamList,
-    RootStackRoutes,
-    ScreenHeader,
-    StackToStackCompositeNavigationProps,
-} from '@suite-native/navigation';
+import { ScreenHeader } from '@suite-native/navigation';
 import { TREZOR_SUPPORT_FW_REVISION_CHECK_FAILED_MOBILE_URL } from '@trezor/urls';
 
 import { DeviceCompromisedModalContent } from './DeviceCompromisedModalContent';
 
 const supportUrlWithChat = `${TREZOR_SUPPORT_FW_REVISION_CHECK_FAILED_MOBILE_URL}#open-chat`;
 
-type NavigationProp = StackToStackCompositeNavigationProps<
-    RootStackParamList,
-    RootStackRoutes.AppTabs,
-    RootStackParamList
->;
+type FirmwareAuthenticityCheckFailModalContentProps = {
+    onCloseRedirect: () => void;
```

Comment:
Looking at the usages of this property, I don't think the naming is right. I'd call it just `onClose` since you cannot know what logic will the caller have there.